src/coreComponents/finiteVolume/TwoPointFluxApproximation.cpp

```diff
@@ -605,6 +613,23 @@ void TwoPointFluxApproximation::addFractureMatrixConnectionsEDFM( MeshLevel & me
   }
 
 }
+void TwoPointFluxApproximation::computeConnectivityIndex( ElementRegionManager & elemManager, 
+                                                          NodeManager & nodeManager, 
+                                                          EmbeddedSurfaceSubRegion & embeddedSurfaceSubRegion ) const
+{
+  // Launch kernel to compute connectivity index of each fractured element.
+  elemManager.forElementSubRegionsComplete< CellElementSubRegion >(
+    [&]( localIndex const, localIndex const, ElementRegionBase &, CellElementSubRegion & subRegion )
+  {
+    auto kernelVariant = createKernel( subRegion.getElementType(), nodeManager, subRegion, embeddedSurfaceSubRegion ); 
+    
+    std::visit( [&] ( auto kernel ) 
+    {
+      using KERNEL_TYPE = decltype( kernel );
+      KERNEL_TYPE::template launchCIComputationKernel< parallelDevicePolicy< >, KERNEL_TYPE >( kernel );
+    }, kernelVariant);  
+  } );
+}
```

Comment:
Maybe these changes should be in their own PR. Moving this here should not have any impact on the results.